src/ShalekKavy.Api/Data/Beverages.json

```diff
@@ -0,0 +1,152 @@
+[
+  {
+    "id": "1",
```

Comment:
The reason why you dont need configuration for the unique keys of your entity is because you are manually adding those in. If you try to add another entity with id 1, it will work but you won't have unique items in the table anymore. Try to define EntityConfigurations so you have a clear definition of what entity constraints your objects are supposed to have. 
https://docs.microsoft.com/en-us/ef/core/modeling/#grouping-configuration